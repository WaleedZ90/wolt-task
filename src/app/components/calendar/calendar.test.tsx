import { Calendar } from ".";
import React from "react";
import renderer from 'react-test-renderer';

describe('<Calendar />', () => {
    it('renders without crashing', () => {
        const component = renderer.create(<Calendar />);
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    })
})