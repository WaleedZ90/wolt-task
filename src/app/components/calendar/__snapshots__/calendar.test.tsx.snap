// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Calendar /> renders without crashing 1`] = `
<article
  className="calendar"
>
  <header>
    <i
      className="far fa-clock header-icon"
    />
    <h1
      className="header-text"
    >
      Opening hours
    </h1>
  </header>
  <section
    className="calendar-section"
  />
</article>
`;
